cogs/moderation.py

```diff
@@ -102,6 +186,45 @@ async def promote(self, ctx, member: discord.Member, role: discord.Role):
         dm_embed.set_footer(text="Promotion Command.")
         await member.send(embed=dm_embed)
 
+    async def _basic_cleanup_strategy(self, ctx, search):
+        count = 0
+        async for msg in ctx.history(limit=search, before=ctx.message):
+            if msg.author == ctx.me:
+                await msg.delete()
+                count += 1
+        return {'Bot': count}
+
+    async def _complex_cleanup_strategy(self, ctx, search):
+        prefixes = tuple(self.client.get_guild_prefixes(ctx.guild))  # thanks startswith
+
+        def check(m):
+            return m.author == ctx.me or m.content.startswith(prefixes)
```

Comment:
You've changed the `m` variable in lines 82, 86, 95, but not here and on line 204, these need to be changed too

Reply:
ok on it.